Sources/SwipeTransition/SwipeBack/SwipeBackController.swift

```diff
@@ -12,7 +12,7 @@ import UIKit
 public final class SwipeBackController: NSObject {
     public var onStartTransition: ((UIViewControllerContextTransitioning) -> Void)?
     public var onFinishTransition: ((UIViewControllerContextTransitioning) -> Void)?
-    private var isFirstPageOfPageViewController: (() -> Bool)?
+    private var shouldBeginSwipeTransition: ((UIGestureRecognizer?) -> Bool)?
```

Comment:
UIGestureRecognizer does not seem to need to be Optional.
```suggestion
    private var shouldBeginSwipeTransition: ((UIGestureRecognizer) -> Bool)?
```